[2024-07-19T19:39:32.947+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-19T19:39:32.974+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_fill_f_101_round_f.end_process manual__2024-07-19T19:38:54.682703+00:00 [queued]>
[2024-07-19T19:39:32.985+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_fill_f_101_round_f.end_process manual__2024-07-19T19:38:54.682703+00:00 [queued]>
[2024-07-19T19:39:32.986+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-19T19:39:33.000+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): end_process> on 2024-07-19 19:38:54.682703+00:00
[2024-07-19T19:39:33.006+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'run_fill_f_101_round_f', 'end_process', 'manual__2024-07-19T19:38:54.682703+00:00', '--job-id', '630', '--raw', '--subdir', 'DAGS_FOLDER/—Åreate_and_fill_f101_round_f.py', '--cfg-path', '/tmp/tmpvsagt3ui']
[2024-07-19T19:39:33.008+0000] {standard_task_runner.py:91} INFO - Job 630: Subtask end_process
[2024-07-19T19:39:33.030+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=18153) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-07-19T19:39:33.032+0000] {standard_task_runner.py:63} INFO - Started process 18154 to run task
[2024-07-19T19:39:33.053+0000] {task_command.py:426} INFO - Running <TaskInstance: run_fill_f_101_round_f.end_process manual__2024-07-19T19:38:54.682703+00:00 [running]> on host 7c7e02e57379
[2024-07-19T19:39:33.131+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_fill_f_101_round_f' AIRFLOW_CTX_TASK_ID='end_process' AIRFLOW_CTX_EXECUTION_DATE='2024-07-19T19:38:54.682703+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-19T19:38:54.682703+00:00'
[2024-07-19T19:39:33.132+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-19T19:39:33.150+0000] {base.py:84} INFO - Using connection ID 'postgres_default' for task execution.
[2024-07-19T19:39:33.171+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-19T19:39:33.172+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-19T19:39:33.180+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=run_fill_f_101_round_f, task_id=end_process, run_id=manual__2024-07-19T19:38:54.682703+00:00, execution_date=20240719T193854, start_date=20240719T193932, end_date=20240719T193933
[2024-07-19T19:39:33.207+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-19T19:39:33.224+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-19T19:39:33.226+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
